wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -0,0 +1,145 @@
+The 4th CJB osu!mania 4K Tournament
+=========================
+
+![CJB 4K 4th](banner.png)
+
+**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, the fourth one-on-one osu!mania 4K specified Chinese regional tournament which is hosted by CJB team.
```

Comment:
`**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, the fourth one-on-one osu!mania 4K specified Chinese regional tournament which is hosted by CJB team.` ->
`**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_** is the fourth one-on-one osu!mania 4K specified Chinese regional tournament hosted by CJB team.`